packages/opensrp-plans/src/components/ListView/utils.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
@p-netm @kahummer should we getting the strings being displayed from a constants/lang module?

Reply:
I'd say both constants and lang file since we have rendered text and the other internal texts